lib/src/picture_provider.dart

```diff
@@ -331,18 +331,22 @@ abstract class PictureProvider<T, U> {
   /// The color filter to apply to the picture, if any.
   final ColorFilter? colorFilter;
 
-  /// The default color applied to SVG elements that inherit the color property.
+  /// The default theme used when parsing SVG elements.
   @visibleForTesting
-  Color? get currentColor => _currentColor;
-  Color? _currentColor;
+  SvgTheme? get theme => _theme;
+  SvgTheme? _theme;
 
-  /// Sets the [_currentColor] to [color].
-  set currentColor(Color? color) {
-    if (_currentColor == color) {
+  /// Sets the [_theme] to [theme].
+  set theme(SvgTheme? theme) {
+    // Fallback to the default theme if null is given.
+    theme = theme ?? const SvgTheme();
```

Comment:
Would this be correct?

Reply:
I think we should just make the setter non-nullable then, right?

Reply:
(as in, make the whole property non nullable).

Also, please update the doc string - we should explain what a theme is and how it's used here.

Reply:
Good point, updated.